src/main/java/seedu/programmer/model/student/UniqueStudentList.java

```diff
@@ -41,15 +41,81 @@
     public boolean contains(Student toCheck) {
         requireNonNull(toCheck);
         return internalList.stream().anyMatch(toCheck::isSameStudent);
+
+    }
+
+    /**
+     * Returns true if the list contains an equivalent student as the given argument.
+     */
```

Comment:
Should documentation include the parameters as well?

Reply:
i have updated the method description, please take a look